sub/lavc_conv.c

```diff
@@ -82,14 +83,23 @@ struct lavc_conv *lavc_conv_create(struct mp_log *log,
     avctx = avcodec_alloc_context3(codec);
     if (!avctx)
         goto error;
-    if (mp_set_avctx_codec_headers(avctx, mp_codec) < 0)
+    if (mp_set_avctx_codec_headers(avctx, sd->codec) < 0)
         goto error;
 
     priv->avpkt = av_packet_alloc();
     priv->avpkt_vtt = av_packet_alloc();
     if (!priv->avpkt || !priv->avpkt_vtt)
         goto error;
 
+    switch (codec->id) {
+    case AV_CODEC_ID_DVB_TELETEXT:;
```

Comment:
stray `;`

Reply:
good catch, no longer needed after moving the code.